pkg/bootkube/status.go

```diff
@@ -24,7 +25,7 @@ const (
 	doesNotExist = "DoesNotExist"
 )
 
-func WaitUntilPodsRunning(pods []string, timeout time.Duration) error {
+func WaitUntilPodsRunning(pods []string, timeout time.Duration, selfHostedEtcd bool) error {
```

Comment:
I could go either way, separate function might be a bit cleaner - but no strong feelings here.